fineract-provider/build.gradle

```diff
@@ -1,4 +1,5 @@
-/**
+targetCompatibility = JavaVersion.VERSION_16
```

Comment:
Please remove this! In the top level build.gradle, it is already configured to marke compatibility to JavaVersion.VERSION_17